uclasm/matching/filters/stats_filter.py

```diff
@@ -0,0 +1,19 @@
+from ..matching_utils import feature_disagreements
+import numpy as np
+
+def stats_filter(smp):
+    """Filtering based on statistics.
+
+    Parameters
+    ----------
+    smp : MatchingProblem
+        A subgraph matching problem on which to compute nodewise cost bounds.
+    """
+    disagreements = feature_disagreements(smp.tmplt.in_out_degrees,
+                                          smp.world.in_out_degrees)
+
+    is_cand = np.logical_and(disagreements <= smp.global_cost_threshold,
```

Comment:
Just do a min between the two cost thresholds and compare to the lower of the two